### DISCUSSION TITLE
How to save one snapshot of every stream and get vods list date wise from api

### DISCUSSION THREAD

**Discussion by rahul78275** [ant-media/Ant-Media-Server#5562] [General]
Hello Team

I have a "raspberry PI 4" device which publish stream to ant media server, and these streams are automatically recorded. I want to auto save one snapshot of every published stream which should return along with VOD link in api response. 

Thank you

**Comment by lastpeony** [comment]
What about using a user defined script?
Run the script after recording is ready on AMS. Implement your desired logic in script
https://antmedia.io/docs/guides/recording-live-streams/user-defined-scripts/
